File: 69/69A.py
n=int(input())
a=0
b=0
c=0
for i in range(n):
    x,y,z=[int(i) for i in input().split()]
    a=a+x
    b=b+y
    c=c+z
if a==0 and b==0 and c==0:
    print("YES")
else:
    print("NO")

File: 69/69A.c
#include<stdio.h>
int main(){
    int a,b,c,t,i;
    scanf("%d", &t);
    int x=0;
    int y=0;
    int z=0;
    for (i=1;i<t+1;++i){
    scanf("%d %d %d", &a, &b, &c);
    x=x+a;
    y=y+b;
    z=z+c;
    }
    if (x==0 && y==0 && z==0){
        printf("YES");
    }
    else{
        printf("NO");
    }
 
    return 0;
}
